### ISSUE TITLE
Android Tablet "spinner" date picker shows calendar too

### ISSUE THREAD

**Issue by Andrea-Arguello** [react-native-datetimepicker/datetimepicker#481]
# Bug report
## Summary

On Tablet emulator, date picker on "spinner" mode will still showcase calendar picker, even after modifying the project's android.

For context: I am using the DateTimePicker as a Field component in redux-form, and have upgraded to the latest version. This issue does not occur on phone emulators.

## Steps to reproduce

1. Create a project using DateTimePicker
2. Add to ```android/app/src/main/res/values/styles.xml``` the following:
```
<style name="SpinnerDatePickerDialog" parent="android:Theme.Material.Light.Dialog">
    <item name="android:datePickerStyle">@style/MyDatePicker</item>
  </style>


  <style name="MyDatePicker" parent="android:Widget.Material.DatePicker">
    <item name="android:calendarViewShown">false</item>
    <item name="android:datePickerMode">spinner</item>
 </style>
```
3. Run app on Android Tablet emulator

Describe what you expected to happen:

1. Only spinner picker would be displayed

What happened:
<img width="856" alt="image" src="https://user-images.githubusercontent.com/31134106/123506247-71b64400-d620-11eb-9cfd-ee0e2b25830d.png">

## Environment info

`npx react-native info` output:

```bash
System:
    OS: macOS 11.4
    CPU: (8) x64 Intel(R) Core(TM) i5-8257U CPU @ 1.40GHz
    Memory: 63.39 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 10.24.0 - ~/.nvm/versions/node/v10.24.0/bin/node
    Yarn: 1.22.4 - /usr/local/bin/yarn
    npm: 6.14.11 - ~/.nvm/versions/node/v10.24.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 14.5, DriverKit 20.4, macOS 11.3, tvOS 14.5, watchOS 7.4
  IDEs:
    Android Studio: 3.6 AI-192.7142.36.36.6392135
    Xcode: 12.5/12E262 - /usr/bin/xcodebuild
  npmPackages:
    react: ~16.9.0 => 16.9.0 
    react-native: ~0.61.4 => 0.61.5
```

datetimepicker version: 3.5.2

Android version: 10.0 (API 29)

**Comment by Andrea-Arguello** [comment]
I noted on the libraries ```android/res/values/styles.xml``` that it only has explicitly declared the SpinnerTimePickerDialog, with target API set to Lollipop.

I've seen that there is an [issue](https://stackoverflow.com/questions/27212267/calendarviewshown-is-not-being-honored-on-android-5-0-lollipop) in Lollipop where, if calendarViewShown and datePickerMode are not set (to "false" and "spinner", respectively), the dialog will show the calendar view, and wondered then, why is the datePicker not explicitly declared in the styles.xml file? And furthermore, why is the target API Lollipop?

**Comment by vonovak** [comment]
:tada: This issue has been resolved in version 6.2.1 :tada:

If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! :rocket:
